### ISSUE TITLE
search with alternate titles

### ISSUE THREAD

**Issue by alidan** [CarlosEsco/Neko#1119]
### Describe your suggested feature

I am optimizing my tachiyomi and neko so if I am going to read it on mangadex its neko, this involves combing through my my tachi manga, and searching them in neko, several times Iput in the correct name, but because it was an alternate title, it wasn't found, which is a bit frustrating. it would be nice if what i'm typing happens to be an alternate title is also searched as well. 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the newest version **[Latest](https://github.com/CarlosEsco/Neko/releases/latest)**.
- [X] I have checked through the app settings for my feature.
- [X] I will fill out all of the requested information in this form.

**Comment by CarlosEsco** [comment]
what you are asking for is impossible.  
The search results returned when browsing is from the Dex Api directly.  I don't have a local copy of all titles of all manga to do some alternative search.

 Checking if search the full name in your example their search is fine. 
![image](https://user-images.githubusercontent.com/2092019/188893546-69027d9b-90a1-4880-b350-ebfd156ff836.png)

So my best guess in your above example is senior is just to vague of a search term (note: switching to sort by highest rated and searching senior worked fine)

**Comment by alidan** [comment]
on browse, it does find it with senior, though it's more toward the bottom.

just so i'm clear on what i'm asking about, there is no way to parse the alternate titles and add them to the library search, it is only able to find the given title I choose if I search it that way?

**Comment by CarlosEsco** [comment]
Oh you want the search in the library to check the alternative titles?  You were talking about adding them from Tachi so I assumed it was browse search

**Comment by alidan** [comment]
Yes, part of what I was doing was having tachi open on the left side, neko on the right, and moving everything mangadex readable to neko because mangadex is just that much better than any other option, and neko is better for reading that tachi. 

tachi had almost everything pulling from mangakalot and many times they would be different titles, so I would search the title in neko and not get a result, thinking that this was something I picked up in mangadexes offline period, I went there to add it, but it also didn't show up, I normally have it hide things already in library, it took a bit of time for 2 + 2 to register, and that's when I found out that the alternate titles did not parse in the library.

Thanks, this will be helpful for when I want to find something but I dont have the name Im searching for it under as its main title.
